YouTubePlayer/YouTubePlayer/YouTubePlayer.swift

```diff
@@ -181,8 +189,12 @@ public class YouTubePlayerView: UIView, UIWebViewDelegate {
     }
 
     private func evaluatePlayerCommand(command: String) {
+        queryPlayerCommand(command)
+    }
+    
+    private func queryPlayerCommand(command: String) -> String? {
```

Comment:
How do you feel about rolling these back into one command and simply ignoring the return value for things like play/pause/etc.?